src/services/shims.ts

```diff
@@ -546,7 +547,7 @@ export class LanguageServiceShimHostAdapter implements LanguageServiceHost {
             return JSON.parse(diagnosticMessagesJson);
         }
         catch (e) {
-            this.log(e.description || "diagnosticMessages.generated.json has invalid JSON format");
+            this.log("diagnosticMessages.generated.json has invalid JSON format");

```

Comment:
This one is what I'd call the only "legitimate" bug; there's no way `e` has a `description` property here; this function just try/catches `JSON.parse`.